modules/ding_debt_easy_reports/views/ding_debt_easy_reports.views.inc

```diff
@@ -0,0 +1,170 @@
+<?php
+/**
+ * @file
+ * Adds views support for showing reports.
+ */
+
+/**
+ * Implements hook_views_data().
+ */
+function ding_debt_easy_views_data() {
+  $data = [];
+
+  $data['ding_debt_easy']['table']['group'] = 'Ding Debt Easy';
+  $data['ding_debt_easy']['table']['base'] = [
+    'field' => 'id',
+    'title' => 'Ding Debt Easy',
+    'help' => t('Nets Easy payment information.'),
+  ];
+
+  $data['ding_debt_easy']['id'] = [
+    'title' => t('Row id'),
+    'help' => t('The consent record id.'),
+    'field' => [
+      'handler' => 'views_handler_field_numeric',
+    ],
+    'sort' => [
+      'handler' => 'views_handler_sort',
+    ],
+    'filter' => [
+      'handler' => 'views_handler_filter_numeric',
+    ],
+  ];
+
+  $data['ding_debt_easy']['order_id'] = [
+    'title' => t('Order ID'),
+    'help' => t('Local order id.'),
+    'field' => [
+      'handler' => 'views_handler_field',
+      'click sortable' => TRUE,
+    ],
+    'sort' => [
+      'handler' => 'views_handler_sort',
+    ],
+    'filter' => [
+      'handler' => 'views_handler_filter',
+    ],
+  ];
+
+  $data['ding_debt_easy']['payment_id'] = [
+    'title' => t('Payment ID'),
+    'help' => t('Payment ID at the gateway.'),
+    'field' => [
+      'handler' => 'views_handler_field',
+    ],
+    'sort' => [
+      'handler' => 'views_handler_sort',
+    ],
+    'filter' => [
+      'handler' => 'views_handler_filter',
+    ],
+  ];
+
+  $data['ding_debt_easy']['charge_id'] = [
+    'title' => t('Charge ID'),
+    'help' => t('Charge ID at the gateway.'),
+    'field' => [
+      'handler' => 'views_handler_field',
+      'click sortable' => TRUE,
+    ],
+    'sort' => [
+      'handler' => 'views_handler_sort',
+    ],
+    'filter' => [
+      'handler' => 'views_handler_filter',
+    ],
+  ];
+
+  $data['ding_debt_easy']['provider_ids'] = [
+    'title' => t('Provider IDs'),
+    'help' => t('Provider IDs at the provider.'),
+    'field' => [
+      'handler' => 'views_handler_field',
+      'click sortable' => TRUE,
+    ],
+    'sort' => [
+      'handler' => 'views_handler_sort',
+    ],
+    'filter' => [
+      'handler' => 'views_handler_filter',
+    ],
+  ];
+
+  // The amount should not be numeric as features has an issue with the
+  // separator value when set to "none". So it fald back to "," as separator
+  // which make ever value over 1000 to 1,000 with then aging is divide with 100
+  // to make into kr. which the endes up being 0.01 and not 10.00 as expected.
+  $data['ding_debt_easy']['amount'] = [
```

Comment:
I am not sure if we are talking thousands separator or decimal separator. If we are talking thousands separator then using `.` might be fine.

Reply:
Have been testet with different field type as this is what worked correctly in all cases.